ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -2320,6 +2320,58 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure the Hide IP Address in Safari is Enabled (Optional)
+  platforms: macOS
+  platform: darwin
+  description: |
+    Public (Routable) IP addresses can be used to track people to their current location, including
+    home and business addresses. While a valid IP addess is necessary to load the site the valid
+    address does not need to be provided to known trackers and should be hidden.
+  resolution: |
+    Perform the following steps to set Safari whether or not to hide IP addresses from trackers:
+    1. Open Safari
+    2. Select Safari from the menu bar
+    3. Select Settings
+    4. Select Privacy
+    5. Set Hide IP address from trackers to Enabled
+  query: |
+    SELECT 1 WHERE NOT EXISTS(SELECT 1 FROM plist WHERE path LIKE
+    "/Users/%/Library/Containers/com.apple.Safari/Data/Library/Preferences/com.apple.Safari.plist"
+    AND key = "WBSPrivacyProxyAvailabilityTraffic" AND (value >> 2) & 1 == 0);
```

Comment:
Apart from checking that all users do not have the feature disabled, should we also check that all of them have the setting enabled? (Similar in the disabled check)